cms/tests/test_class_properties.py

```diff
@@ -0,0 +1,31 @@
+from cms.admin.forms import AdvancedSettingsForm, ChangePageForm
+from cms.models.contentmodels import PageContent
+from cms.models.pagemodel import Page
+from cms.test_utils.testcases import CMSTestCase
+
+
+class PagePropsMovedToPageContentTests(CMSTestCase):
+
+    def test_moved_constants(self):
+        """test constants moved from Page class"""
+        self.assertFalse(hasattr(Page, "LIMIT_VISIBILITY_IN_MENU_CHOICES"))
+        self.assertFalse(hasattr(Page, "TEMPLATE_DEFAULT"))
+        self.assertFalse(hasattr(Page, "X_FRAME_OPTIONS_INHERIT"))
+        self.assertFalse(hasattr(Page, "X_FRAME_OPTIONS_DENY"))
+        self.assertFalse(hasattr(Page, "X_FRAME_OPTIONS_SAMEORIGIN"))
+        self.assertFalse(hasattr(Page, "X_FRAME_OPTIONS_ALLOW"))
+        self.assertFalse(hasattr(Page, "X_FRAME_OPTIONS_CHOICES"))
+
+        """test Page class constants present in PageContent class"""
+        self.assertTrue(hasattr(PageContent, "LIMIT_VISIBILITY_IN_MENU_CHOICES"))
+        self.assertTrue(hasattr(PageContent, "TEMPLATE_DEFAULT"))
+        self.assertTrue(hasattr(PageContent, "X_FRAME_OPTIONS_CHOICES"))
+
+    def test_moved_attributes(self):
```

Comment:
Hi @theShinigami ! Thanks for adding the test. Two thingws come to my mind:
1. I think it's important that the test checks of the change form contains the field. This test will also pass without the fix. The easiest way to check might be to iterate through the `PageChangeForms`'s fieldsets to make sure it is included. 
2. Should we test this for all fields of `PageContent`? It seems to me the X frame options was just accidentally forgotten. Maybe next time some other field is forgotten?